### PR TITLE
add windows process start_time

### LINKED ISSUE #11641: Add windows process start_time
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| v4.4.0|syscollector | Agent | Sources | Windows |

## Description

Hi,

This is a PR https://github.com/wazuh/wazuh/pull/11640 to add windows process start_time.
Please review it.

Thanks.

### PR DESCRIPTION
**Pull request by LubinLew**
|Related issue|
|---|
| Closes #11641 |

## Description

add windows process start_time

## Tests

- Compilation without warnings in every supported platform
  - [ ] Linux
  - [x] Windows
  - [ ] MAC OS X
- [x] Source installation
- [ ] Package installation
- [ ] Source upgrade
- [ ] Package upgrade
- [ ] Review logs syntax and correct language
- [ ] QA templates contemplate the added capabilities

- Memory tests for Linux
  - [ ] Scan-build report
  - [ ] Coverity
  - [ ] Valgrind (memcheck and descriptor leaks check)
  - [ ] Dr. Memory
  - [ ] AddressSanitizer
- Memory tests for Windows
  - [ ] Scan-build report
  - [ ] Coverity
  - [ ] Dr. Memory
- Memory tests for macOS
  - [ ] Scan-build report
  - [ ] Leaks
  - [ ] AddressSanitizer

- [x] Retrocompatibility with older Wazuh versions
- [ ] Working on cluster environments
- [ ] Configuration on demand reports new parameters
- [ ] The data flow works as expected (agent-manager-api-app)
- [ ] Added unit tests (for new features)
- [ ] Stress test for affected components

- Decoder/Rule tests
  - [ ] Added unit testing files ".ini"
  - [ ] runtests.py executed without errors

### CODE REVIEW

**Review by jnasselle — CHANGES_REQUESTED** [review]
Hi @LubinLew,

Thanks again for your contributions!Amazing job! Current PR LGTM, but the branch was created from 4.3 branch and trying to be merged to master. This should be changed and created from master.

Regards,
Nico

**Comment by LubinLew** [comment]
Hi @jnasselle ,

I Merged master into this PR branch, Can this meet the requirements?
I used the master branch at first, but there will be sqlite3 link errors, so I used 4.3 branch.

**Comment by Dwordcito** [comment]
@LubinLew 
if you run (from the src folder) `rm -rf external/* && make deps TARGET=agent && make TARGET=agent` does the problem persist?
between different versions of wazuh the external dependencies vary, which can cause a link problem.

**Comment by LubinLew** [comment]
@Dwordcito 

It works after cleaning the external dependencies.

Thanks.

**Comment by jnasselle** [comment]
## End-to-end testing :green_circle: 
kibana
conversion
processexplorer

**Review by jnasselle — APPROVED** [review]

File: src/data_provider/src/sysInfoWin.cpp [hunk 1]
```diff
@@ -76,6 +77,12 @@ class SysInfoProcess final
             return ret;
         }
 
+        ULONGLONG creationTime() const
+        {
+            constexpr auto WINDOWS_UNIX_EPOCH_DIFF_SECONDS { 11644473600ULL };
```
Comment (RamiroRD) [inline src/data_provider/src/sysInfoWin.cpp:82]: Maybe we could leave a comment explaining where this was taken from

**Review by HanesSciarrone — APPROVED** [review]

**Review by jnasselle — APPROVED** [review]

**Review by HanesSciarrone — APPROVED** [review]

**Review by Dwordcito — APPROVED** [review]

**Pull request merged**